<!DOCTYPE html>
<html>
<title>Camila Farias Amorim, MS., Ph.D</title>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=3">
<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
<link rel="shortcut icon" href="images/logo/heatmap.png">
<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
<style>
body, h1,h2,h3,h4,h5,h6 {font-family: "Montserrat", sans-serif}
.mySlides {display: none}
.bgimg {
  background-position: center;
  background-repeat: no-repeat;
  background-size: cover;
  background-image: url('/images/others/pelourinho.jpeg');
  min-height: 100%;
}
img {
  float: left, right;
}
</style>
<body>

<!-- Sidebar with image -->
<nav class="w3-sidebar w3-hide-medium w3-hide-small" style="width:40%">
  <div class="bgimg"></div>
</nav>

 Hidden Sidebar (reveals when clicked on menu icon)-->
<nav class="w3-sidebar w3-black w3-animate-right w3-xxlarge" style="display:none;padding-top:150px;right:0;z-index:2" id="mySidebar">
  <a href="javascript:void(0)" onclick="closeNav()" class="w3-button w3-black w3-xxxlarge w3-display-topright" style="padding:0 12px;">
    <i class="fa fa-remove"></i>
  </a>
  <div class="w3-bar-block w3-center">
    <a href="#" class="w3-bar-item w3-button w3-text-grey w3-hover-black" onclick="closeNav()">Home</a>
    <a href="#about" class="w3-bar-item w3-button w3-text-grey w3-hover-black" onclick="closeNav()">About Me</a>
    <a href="#currentrole" class="w3-bar-item w3-button w3-text-grey w3-hover-black" onclick="closeNav()">Current role</a>
    <a href="#keywords" class="w3-bar-item w3-button w3-text-grey w3-hover-black" onclick="closeNav()">Keywords</a>
    <a href="#publications" class="w3-bar-item w3-button w3-text-grey w3-hover-black" onclick="closeNav()">Academic publications</a>
    <a href="#cv" class="w3-bar-item w3-button w3-text-grey w3-hover-black" onclick="closeNav()">CV</a>
    <a href="#footer" class="w3-bar-item w3-button w3-text-grey w3-hover-black" onclick="closeNav()">Links and Contact</a>
  </div>
</nav>

<!-- Page Content -->
<div class="w3-main w3-padding-large" style="margin-left:40%">

  <!-- Menu icon to open sidebar -->
  <span class="w3-button w3-top w3-white w3-xxlarge w3-text-grey w3-hover-text-black" style="width:auto;right:0;" onclick="openNav()"><i class="fa fa-bars"></i></span>

  <!-- Header -->
  <header class="w3-container w3-center" style="padding:20px 16px" id="home">
    <h2><b>Camila Farias Amorim, Ph.D.</b></h2>
    <p>Immunology - Clinical/Translational Medicine</p>
    <p>Computational Biology - Bioinformatics</p>
    <p>OMICS - Molecular Biology</p>
  </header>

    <!-- About and photo -->
  <div class="w3-content w3-justify w3-text-grey w3-padding-32" id="about">
    <h2>About Me</h2>
    <hr class="w3-opacity">    
    <p>Hello! Welcome to my website.</p>
    <p>Computational Biology & Immunology PhD scientist specialized working in Omics, molecular profiling, and AI-enabled biomarker discovery to support allogeneic iPSC-derived cell therapies, spanning immune (CAR-T/CAR-NK) and non-immune (SC–derived pancreatic islet) programs for oncology and Type 1 Diabetes.  </p>
    <p>Bringing 12+ years of experience across wet- and dry-lab settings, integrating cellular biology, immunopathogenesis, and high-dimensional genomics to enable mechanism-based biomarker discovery, ML predictive modeling, and translational decision-making across preclinical and early clinical development.</p>
    <p>Incorporating AI-assisted analytics, including large language models (LLMs), to support workflow automation, reproducible analysis, scalable data exploration, and hypothesis generation in biomarker and target discovery.</p>
  </div>

  <!-- Current role -->
  <div class="w3-content w3-justify w3-text-grey w3-padding-32" id="currentrole">
    <h2>Current role</h2>
    <hr class="w3-opacity">  
    <p>Currently working on a Principal Scientist role at <a href="https://www.centurytx.com/" target="_blank">Century Therapeutics Inc.</a>,
    <a href="https://www.linkedin.com/in/camila-farias-amorim-ms-phd/" target="_blank">LinkedIn</a></p>  
    <p>Leading the NGS and molecular assay strategies across research, cell therapy manufacturing, and external stakeholder communication. Scope includes study design, assay development (qPCR, flow cytometry, CITE-seq, scRNA-seq, spatial transcriptomics), PK/PD and statistical modeling, and multi-omics data integration to inform product optimization and program strategy.</p>  
    <p>Works horizontally across iPSC biology, immunology, differentiation, translational, manufacturing, and clinical teams, and oversees Omics-focused collaborations with CROs and technology vendors, ensuring experimental rigor, data quality, and cross-study comparability.</p>
    <p>I lead company-wide Research Omics forums to align wet- and dry-lab teams and translate complex NGS datasets into actionable insights. Background includes biomarker and target discovery, predictive modeling, immunotherapies, vaccines, and early clinical biomarker strategy, with prior experience teaching NGS and bioinformatics at the University of Pennsylvania and 20+ peer-reviewed publications.</p>
    <p>Focuses on connecting biological insight, quantitative analysis, and collaborative execution to support rigorous science and patient-relevant outcomes.</p>

      <br><br>

    <p>Avaliable materials:</p>
    <p>1. Omics lead in: <i>iPSC-derived CD4+, CD8+ αβT cells that expand upon antigen stimulation and exhibit in vivo tumor control. 2025. </i><a href="https://www.centurytx.com/wp-content/uploads/Keystone-2025-alpha-beta-T-cells.pdf" target="_blank">Keystone poster</a></p>
   
     <br><br>

    <p>#ComputationalBiology #BiomarkerDiscovery #PredictiveModeling #MolecularProfiling #LLMs #SingleCell #SpatialTranscriptomics #CellTherapy #iPSC #CAR_T #CAR_NK #SCIslets #TranslationalScience #Omics #NGS #CROManagement #DEI</p>
  </div>

  <!-- Keywords -->
  <div class="w3-content w3-justify w3-text-grey w3-padding-32" id="keywords">
    <h2>Keywords</h2>
    <hr class="w3-opacity">    
      high-throughput NGS; AI/LLMs; ML predictive modeling; R; python; C++; bash; Unix/Linux OS; command-line tools; shell scripting; read mapping; Illumina sequencing; DNA; RNA; RNAseq, Dual-RNAseq; single-cell RNAseq, CITE-Seq, 16S-seq, WGS, clinical metadata; Bioconductor; tidyverse; data visualization; linear modeling; machine learning; supervised/unsupervised classification; hierarchical clustering; dimensionality reduction; pattern recognition; git/GitHub; version control; cloud; ShinyApp; datasets; ggplot; statistical inference; pathway analysis; GSEA; GO; GEO NCBI
   </div>   

  <!-- Publications -->
  <div class="w3-content w3-justify w3-text-grey w3-padding-32" id="publications">
    <h2>Academic publications</h2>
    <hr class="w3-opacity">
    <p>A complete list of publications can be found in my
      <a href="https://scholar.google.com/citations?user=6jg00_wAAAAJ&hl=en" target="_blank">Google Scholar Profile</a>
      or more on <a href="https://www.webofscience.com/wos/author/record/2416090" target="_blank">Web of Science</a>. See below my main published projects:</p>
      
      <br><br>

      <h5><p><b> Variable gene expression and parasite load predict treatment outcome in cutaneous leishmaniasis</p></b></h5>
       <p><img src="/images/content/Amorim2019.001.jpeg" class="w3-image" style="float:left;width:280px;height:150px;margin-right:15px;margin-bottom:10px;">
        <i><u>Biological significance:</i></u> Most of the patients infected by <i>L. braziliensis</i> fail the therapy with an anti-parasitic drug, and there are no vaccines for this infection. 
        I discovered that both parasite burden and the cytotoxic effector mechanism from skin CD8+ T cells/NK cells affect the response to therapy and can be used as an early biomarker at the clinic. 
        The immunotherapy targeting this later inflammatory mechanism in experimental models dramatically impacted the control of the disease.</p>
        
        <p><i><u>Bioinformatics:</i></u> This was my first publication merging Immunology and Computational Biology. 
        I investigated a human skin, lesional RNA-seq dataset from Brazilian cutaneous leishmaniasis patients and developed a workflow focused on genes with a highly variable expression between samples. 
        In this study, R statistical programming language, dual-RNAseq mapping (quantifying both human and parasite reads from the samples), 
        and a combination of strategies to reduce the dimensionality of the metadata were used to find these genetic biomarkers. All the code involved in reproducing the analysis and images is available in a 
        <a href="https://doi-org.proxy.library.upenn.edu/10.24433/CO.5903311.v1" target="_blank">CodeOcean capsule</a>.</p>
       <p><i>Farias Amorim et al. 2019, Science Translational Medicine, <a href="https://stm.sciencemag.org/content/11/519/eaax4204?rss=1" target="_blank">DOI: 10.1126/scitranslmed.aax4204</a></i></p>
       <p><i>Code capsule: <a href="https://codeocean.com/capsule/6534016/tree/v1" target="_blank"> CodeOcean</a></i></p>
       <div class='altmetric-embed' data-badge-type='donut' data-doi="10.1126/scitranslmed.aax4204"></div>

       <br><br>
       
       <h5><p><b> Multi-omic profiling of cutaneous leishmaniasis infections reveals microbiota-driven mechanisms underlying disease severity</p></b></h5>
       <p><img src="/images/content/Amorim2023.png" class="w3-image" style="float:left;width:730px;height:290px;margin-right:15px;margin-bottom:10px;">
        <i><u>Biological significance:</i></u> The development of the lesion caused by the <i>Leishmania braziliensis</i> infection in humans is influenced by several factors. 
        I collaborated with a big team of Brazilian clinicians and experimentalist bench scientists and analyzed a multi-omics dataset from those patients. 
        We discovered that the skin microbiome strongly contributes to the severity of the disease, delayed healing, and unrevealed IL-1b as a major inflammatory mediator behind this association. 
        This study's results support targeted immunotherapies and bacterial therapies for this parasitic infection.</p>
        
        <p><i><u>Bioinformatics:</i></u> I have reduced the dimensionality, integrated and analyzed a multi-omics dataset of L. braziliensis patients (skin RNA-seq, 16S-seq, bacterial WGS, qPCRs, and clinical metadata). 
        I developed a computational workflow with the R programming language based on multivariate linear regression analysis and machine learning methods to identify elements contributing to the worse clinical outcome. 
        <p><i>Farias Amorim et al. 2023, Science Translational Medicine, <a href="https://www.science.org/doi/10.1126/scitranslmed.adh1469" target="_blank">DOI: 10.1101/2023.02.02.23285247</a></i></p>
       <div class='altmetric-embed' data-badge-type='donut' data-doi="10.1126/scitranslmed.adh1469"></div>
       
       
       <br><br>

       <h5><p><b> Localized skin inflammation during cutaneous leishmaniasis drives a chronic, systemic IFN-γ signature</p></b></h5>
       <p><img src="/images/content/Amorim2021.png" class="w3-image" style="float:right;width:470px;height:170px;margin-left:15px;margin-bottom:10px;">
        <i><u>Biological significance:</i></u> In this study, we were interested in evaluating whether the localized skin infection caused by the <i>Leishmania braziliensis</i> impacts patients' systemic immunity. 
        We discovered a monocyte-Interferon-gamma and CD8+ T cells cytotoxic (CTL) transcriptional inflammation at the site of skin infection and systemically (lesion and peripheral blood RNA-seq analysis). 
        Moreover, when we compared the blood from those patients with subjects with other diseases (tuberculosis and malaria), we identified the transcriptional CTL signature as a unique circulatory feature of this parasitic disease. 
        We amplified our knowledge about this localized skin condition: cutaneous leishmaniasis. We raised the question of how these systemic responses might influence the response to other infections and the overall health of these individuals.

       <p><i><u>Bioinformatics:</i></u> I used linear modeling, statistical inference, and data visualization strategies to profile the systemic transcriptome of patients with cutaneous leishmaniasis and revealed the gene expression patterns associated with clinical variables and outcomes. 
        I also downloaded publicly available blood RNA-seq datasets from patients with other infections and compared their transcriptomes to identify disease-specific data patterns.</p>
       
       <p><i>Farias Amorim et al. 2021, PLoS NTDs, <a href="https://journals.plos.org/plosntds/article?id=10.1371/journal.pntd.0009321" target="_blank">DOI: 10.1371/journal.pntd.0009321</a></i></p>
       <div class='altmetric-embed' data-badge-type='donut' data-doi="10.1371/journal.pntd.0009321"></div>
       
       <br><br>

       <h5><p><b> Teaching: An Open-Source Toolkit To Expand Bioinformatics Training in Infectious Diseases</p></b></h5>
       <p><img src="/images/content/class.jpeg" class="w3-image" style="float:left;width:380px;height:270px;margin-right:15px;margin-bottom:0px;">
        I was a Teaching Assistant in the <a href="http://diytranscriptomics.com" target="_blank">DIYtranscriptomics.com</a> course led by Dr. Beiting at the University of Pennsylvania.
        As a short description (almost unfair), we teach how to work with transcriptomic data (bulk and single-cell RNA-seq): from installing the necessary tools to writing base/tidy R, using Bioconductor R packages, data modeling, and final next-generation sequencing interpretation of biological results.
        We use examples of real-world datasets in the context of Infectious Diseases to introduce updated multi-omics concepts from the greater area of Computational Biology.
        The course is fully available online, and our team put together a paper gathering everything we have learned in past years.</p>

       <p><i>Berry and Farias Amorim et al. 2021, mBio, <a href="https://journals.asm.org/doi/pdf/10.1128/mBio.01214-21" target="_blank">DOI: 10.1128/mBio.01214-21</a></i></p>
       <p><i>Code capsule: <a href="https://codeocean.com/capsule/1031026/tree/v3" target="_blank"> CodeOcean</a></i></p>
       <div class='altmetric-embed' data-badge-type='donut' data-doi="10.1128/mBio.01214-21"></div>
       
       <br><br>
  </div>

  <!-- CV -->
  <div class="w3-content w3-justify w3-text-grey w3-padding-32" id="cv">
    <h2>Curriculum vitae</h2>
    <hr class="w3-opacity">   
    <p>My <a href="https://drive.google.com/file/d/1U4YFXJfu0llZlIxhPq-WzYo1w8PCRqig/view?usp=sharing" target="_blank">CV</a>. (last update: December 2023).</p> 
    <p>TBD update...</p>
  </div>

  <!-- My picture -->  
  <div class="w3-content w3-center w3-text-grey">
    <hr class="w3-opacity">
    <img src="/images/photo/pic2_copy.png" class="w3-image" style="float:center;width:200px;height:190px;">
  </div>
  <br><br>

  <!-- Links and Contact -->
  <footer class="w3-container w3-padding-64 w3-light-grey w3-center w3-opacity w3-xlarge" style="margin:-24px" id="footer">
    <p class="w3-medium"><b>Contact and Links</b></p>
    <a href="https://twitter.com/amorim_cfa" target="_blank"><i class="fa fa-twitter w3-hover-opacity"></i></a>
    <a href="https://github.com/camilafarias112" target="_blank"><i class="fa fa-github w3-hover-opacity"></i></a>
    <a href="https://www.linkedin.com/in/camila-farias-amorim-ms-phd/" target="_blank"><i class="fa fa-linkedin w3-hover-opacity"></i></a>
    <a href="mailto:camilafarias112@gmail.com" target="_blank"><i class="fa fa-envelope-o"></i></a>
    <p class="w3-medium">Philadelphia, PA, USA</p>
    <p class="w3-medium">© 2026. Camila Farias Amorim. All rights reserved.</p>
    <p class="w3-medium">Using :: <a href="http://jekyllrb.com/" target="_blank">Jekyll</a></p>
  <!-- End footer -->
  </footer>
  
<!-- END PAGE CONTENT and JAVA -->
</div>

<script>
// Open and close sidebar
function openNav() {
  document.getElementById("mySidebar").style.width = "60%";
  document.getElementById("mySidebar").style.display = "block";
}

function closeNav() {
  document.getElementById("mySidebar").style.display = "none";
}
</script>

<script>
// Slideshow Apartment Images
var slideIndex = 1;
showDivs(slideIndex);

function plusDivs(n) {
  showDivs(slideIndex += n);
}

function currentDiv(n) {
  showDivs(slideIndex = n);
}

function showDivs(n) {
  var i;
  var x = document.getElementsByClassName("mySlides");
  var dots = document.getElementsByClassName("demo");
  if (n > x.length) {slideIndex = 1}
  if (n < 1) {slideIndex = x.length}
  for (i = 0; i < x.length; i++) {
    x[i].style.display = "none";
  }
  for (i = 0; i < dots.length; i++) {
    dots[i].className = dots[i].className.replace(" w3-opacity-off", "");
  }
  x[slideIndex-1].style.display = "block";
  dots[slideIndex-1].className += " w3-opacity-off";
}
</script>

</body>
</html>